diffpy/snmf/tests/test_subroutines.py

```diff
@@ -252,7 +252,16 @@ def test_reconstruct_signal(trs):
     actual = reconstruct_signal(trs[0], trs[1])
     assert len(actual) == len(trs[0][0].grid)
 
-trsh = []
+trsh = [([ComponentSignal([0, .25, .5, .75, 1], 2, 0), ComponentSignal([0, .25, .5, .75, 1], 2, 1),
+         ComponentSignal([0, .25, .5, .75, 1], 2, 2)], 1),
+       ([ComponentSignal([0, .25, .5, .75, 1], 2, 0), ComponentSignal([0, .25, .5, .75, 1], 2, 1),
+         ComponentSignal([0, .25, .5, .75, 1], 2, 2)], 0),
+       ([ComponentSignal([0, .25, .5, .75, 1], 3, 0), ComponentSignal([0, .25, .5, .75, 1], 3, 1),
+         ComponentSignal([0, .25, .5, .75, 1], 3, 2)], 2),
+       # ([ComponentSignal([0, .25, .5, .75, 1], 2, 0), ComponentSignal([0, .25, .5, .75, 1], 2, 1),
+        # ComponentSignal([0, .25, .5, .75, 1], 2, 2)], -1),
+]
 @pytest.mark.parametrize('trsh', trsh)
-def test_reconstruct_signal_hessian():
-    assert False
+def test_reconstruct_signal_hess(trsh):
+    actual = reconstruct_signal(trsh[0], trsh[1])
+    assert len(actual) == len(trsh[0][0].grid)
```

Comment:
This seems to be just testing the shape of the array handed back.  Is that your intention?